.changeset/clean-meals-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+"web-fragments": patch
```

Comment:
Note: this is actually a breaking change, but at this stage I think we should always just do patch bumps (to stay in 0.0.x)